enterprise/models.py

```diff
@@ -4055,7 +4055,14 @@ def submit_for_configuration(self, updating_existing_record=False):
             is_sap = True
         else:
             for field in self.base_saml_config_fields:
-                config_data[utils.camelCase(field)] = getattr(self, field)
+                
+                if field == "active":
+                    if not updating_existing_record:
+                        config_data['enable'] = True
```

Comment:
Are we moving forward under the assumption we won't be saving draft configurations?

Reply:
MVP will not implement drafts, however this updated logic doesn't necessarily block us from doing so. This is in our method to send a config to the orchestrator for configuration. If the self service tool is sending a POST for a draft to our API, we’d have to make sure the api view doesn’t submit for configuration, but that's part of the drafts implementation.